tuning/sensorless.md

```diff
@@ -0,0 +1,477 @@
+---
+layout: default
+title: Configuring sensorless homing
+parent: Tuning Guides
+nav_exclude: true
+---
+
+This writeup covers how to set up sensorless homing on an Octopus 1.1 with 2209 drivers. Different drivers and/or MCUs may differ. Adjust accordingly.
+
+Also note that there are example working configurations at the bottom of this guide. Use them as examples based on your MCU and drivers, etc.
+
+{: .note }
+IF YOU DO NOT READ THIS CAREFULLY AND ADJUST THINGS WHEN NEEDED YOU WILL BE SAD. IF YOU USE A PHYSICAL ENDSTOP YOU MUST CHANGE THINGS TO HOME Z TO THE ENDSTOP PIN, NOT THE MIDDLE OF THE BED. YOU HAVE BEEN WARNED!
+
+At the end of this document is a section with working examples you can use as a reference for some common hardware
+
+# Getting started
+
+Since this example revolves around an Octopus 1.1, you need to verify which diag jumpers to install based on your hardware and motor positions. The examples below serve to illustrate the process.
+
+
+1. Power off the printer gracefully
+2. Physically disconnect the X and Y endstop connections from the MCU.
+3. Plug in jumpers on the `DIAG` ports as shown below (**J16** and **J17**). We use these ports since thats the corresponding diag jumper for where our XY steppers are connected.
```

Comment:
```suggestion
3. Plug in jumpers on the `DIAG` ports as shown below (**J16** and **J17**). We use these ports since these are the corresponding diag jumpers for where our XY steppers are connected.
```